perception/point_cloud_object_detection_and_recognition/include/point_cloud_object_detection_and_recognition/pcodar_params.hpp

```diff
@@ -0,0 +1,71 @@
+#pragma once
+
+#include <ros/ros.h>
+
+#include <string>
+#include <vector>
+
+#include <Eigen/Geometry>
+
+namespace pcodar
+{
+
+extern struct pcodar_params
```

Comment:
Should probably be replaced by a dynamic reconfig file (which will generate a header for this struct)